interfaces/Config/templates/config_switches.tmpl

```diff
@@ -321,11 +321,7 @@
                     <div class="field-pair">
```

Comment:
As long as the rating host isn't communicated through the NZB, the host should be easy to set.